edgePy/io/_io.py

```diff
@@ -7,12 +7,55 @@
 
 
 __all__ = [
-    'Importer',
-    'get_dataset_path'
+    'GroupImporter',
+    'DataImporter',
+    'get_dataset_path',
+    'get_open_function'
 ]
 
 
-class Importer(object):
+def get_open_function(filename):
```

Comment:
I think testing for a suffix to infer encoding is fragile. 

Painfully, you can have a gzipped file named `hello.txt`.

I would change the name of the function to `infer_open_function`.

Reply:
Yeah, it's very fragile.  This is just bootstrapping.  I assume 95% of what I'm writing is just a placeholder to get us off the ground as fast as possible, and give us something to frame around.

You're absolutely right, though, this should be replaced with something less fragile.